packages/expo-updates/android/src/main/java/expo/modules/updates/db/dao/UpdateDao.java

```diff
@@ -52,6 +55,10 @@
     return _loadUpdatesForProjectWithStatuses(scopeKey, Arrays.asList(UpdateStatus.READY, UpdateStatus.EMBEDDED, UpdateStatus.DEVELOPMENT));
   }
 
+  public List<UpdateEntity> loadEmbeddedUpdates() {
+    return _loadUpdatesWithStatus(UpdateStatus.EMBEDDED);
+  }
+
```

Comment:
Maybe just do a public `loadUpdatesWithStatus` instead of adding this extra abstraction layer?